source/best-practices/backup.rst

```diff
@@ -1,61 +1,133 @@
 备份与还原
 ==========
 
+:本节贡献者: |田冬冬|\（作者）、
+             |姚家园|\（作者）
+:最近更新日期: 2021-01-30
+:预计阅读时间: 20 分钟
+
+----
+
 **备份！备份！备份！**
 
 尽管不经常发生，但电脑随时可能会坏掉或被盗，电脑硬盘也随时可能会出问题，
-也可能不小心使用一个命令把几个月的工作误删了。所以，备份非常重要，不进行必要的备份
-是十分危险的。
+也可能不小心使用一个命令（如 ``rm *``）把几个月的工作误删了。所以，备份非常重要，不进行必要的备份
+是十分危险的。笔者和一些同行多次体验过数据误删或硬盘损坏又未备份带来的痛苦。
```

Comment:
I think @CovMat mentioned that the average lifetimes of hard drives are about 5 years, due to the frequent disk I/O in scientific research. I think that's a big warning for everyone.

Reply:
done in b345fe6.